contrib/seeds/makeseeds.py

```diff
@@ -48,7 +48,7 @@
     r"25.(0|1|2|99).0|"
     r"26.(0|1|99).0|"
     r"27.(0|1|99).0|"
-    r"28.(0|99).0|"
+    r"28.(0|99).0"
```

Comment:
In 6597e87666daa7fabdf310e9ad39fc8a7c81ceea "makeseeds: fix incorrect regex"

Perhaps put the `|` in the front of every regex rather than the end, to make it easier to update in the future and not accidentally run into this problem again.

Reply:
Updating to take this feedback and https://github.com/bitcoin/bitcoin/pull/31960#issuecomment-2695738356, thanks.

Reply:
Done